code/modules/food_and_drinks/food/foods/ethnic.dm

```diff
@@ -135,35 +135,43 @@
 //	Middle Eastern	//
 //////////////////////
 
-/obj/item/food/snacks/human/kabob
-	name = "-kabob"
-	icon_state = "kabob"
-	desc = "Human meat, on a stick."
+/obj/item/food/snacks/meatkebab
+	name = "meat-kebab"
+	icon_state = "kebab"
+	desc = "Delicious meat, on a stick."
 	trash = /obj/item/stack/rods
 	filling_color = "#A85340"
 	list_reagents = list("nutriment" = 8)
+	goal_difficulty = FOOD_GOAL_NORMAL
 
-/obj/item/food/snacks/monkeykabob
-	name = "meat-kabob"
-	icon_state = "kabob"
-	desc = "Delicious meat, on a stick."
+/obj/item/food/snacks/syntikebab
+	name = "synthimeat-kebab"
+	icon_state = "kebab"
+	desc = "Delicious synthetic meat, on a stick."
+	trash = /obj/item/stack/rods
+	filling_color = "#A85340"
+	list_reagents = list("nutriment" = 8)
+
+/obj/item/food/snacks/human/kebab
+	name = "-kebab"
+	icon_state = "kebab"
+	desc = "Human meat, on a stick."
```

Comment:
```suggestion
/obj/item/food/snacks/human/kebab
	name = "-kebab"
	desc = "Human meat, on a stick."
	icon_state = "kebab"
```
Its written "-kebab" to make it look different from the customizable version?
![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/3b3b0088-ae42-475a-8738-a28a68fe32cf)

Reply:
snacks/human should take the name of player meat and add it in front of the hyphen (e.g. Goobina-kebab).